modules/apps/headless/headless-user-notification/headless-user-notification-impl/src/main/java/com/liferay/headless/user/notification/internal/dto/v1_0/UserNotificationDTOConverter.java

```diff
@@ -73,6 +73,7 @@ public UserNotification toDTO(
 		return new UserNotification() {
 			{
 				dateCreated = new Date(userNotificationEvent.getTimestamp());
+				entryTitle = jsonObject.getString("entryTitle", null);
```

Comment:
please, check first if the element is present on the jsonObject, as made on the `setType`